### DISCUSSION TITLE
Value/entity for current modulation of heatpump

### DISCUSSION THREAD

**Discussion by Th0maz** [emsesp/EMS-ESP32#1283] [Q&A] [answered]
Hey everybody,
I'm quite new to the ems-esp project and bought an interface from bbqkees a couple of weeks ago.
It is already up and running, sending all values to telegraf via MQTT and saving them to an influxdb2.
Now I was hoping for a value, which mirrors the current modulation of my Buderus LWL 196i heatpump.
Is there any value (or should there be any), that tells me if the heatpump is running on 100% or less and what's the current modulation?
There's an entity shown in the web interface called "heatpump output", but that's always zero. 
Regarding to the ems-esp web interface I've got a Hybrid Manager HM 200 in my system.
Can you please help me out with this?
Thanks!
Thomas

PS:
I've been monitoring the power consumption of my heatpump for a while and I'd like to get something similar through the ems-esp gateway.
Power consumption graph (in %):
![grafik](https://github.com/emsesp/EMS-ESP32/assets/12065628/66354f6e-0a00-4620-b35f-c038d4fc734c)

**Comment by proddy** [comment]
which EMS-ESP version are you on?

**Reply by Th0maz** [reply]
3.5.1

**Reply by proddy** [reply, marked as answer]
try https://github.com/emsesp/EMS-ESP32/releases/tag/latest

it has some heatpump additions. not sure if its what you need but always good to be on the leading edge

**Reply by MichaelDvP** [reply]
Yes, compressor speed is fixed in 3.6.1, see #1247,

**Reply by Th0maz** [reply]
Thank you both.
I updated to 3.6.0 today and now see the hpcompspd entity that I can use :)

**Reply by MichaelDvP** [reply]
Use the 3.6.1-dev, it will be released soon and fixes a mqtt issue.
